docs/tutorials/getting-started.md

```diff
@@ -42,7 +42,47 @@ with the tests.
 
 ## Run the test suite
 
-🚧 **TODO**
+#### Redash Setup
+The Redash instance must be setup before running the tests. Start the images ```docker-compose up -d```. This will download the images if you don't have them.
```

Comment:
I would explain that ``docker-compose up -d`` will download the required images and build and started containers in the background. Can you please also add how to stop them?